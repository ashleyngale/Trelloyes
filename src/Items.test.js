import React from 'react';
import ReactDOM from 'react-dom';
import Messages from './Items';
import { createRenderer } from 'react-dom/test-utils';
import Item from './Items.js';

describe('Item component', ()=> {
it('renders without crashing', () => {
  const div = document.createElement('div');
    ReactDOM.render(<item />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('renders amount of card clicks with no clicks', ()=> {
  const tree = renderer 
  .create(<item name="items" item={0}/>)
  .toJSON();
  expected(tree).toMatchSnapSHot();
});