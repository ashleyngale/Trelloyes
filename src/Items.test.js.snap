import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Item from './Items.js';

describe('Item component', ()=> {
it('renders without crashing', () => {
  const div = document.createElement('div');
    ReactDOM.render(<Item />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('renders amount of card clicks with no clicks', ()=> {
  const tree = renderer
  .create(<Item name="items" Item={0}/>)
  .toJSON();
  expected(tree).toMatchSnapSHot();
});
});